django/db/utils.py

```diff
@@ -115,7 +115,7 @@ def load_backend(backend_name):
         import django.db.backends
         builtin_backends = [
             name for _, name, ispkg in pkgutil.iter_modules(django.db.backends.__path__)
-            if ispkg and name not in {'base', 'dummy', 'postgresql_psycopg2'}
```

Comment:
not even postgresql?

Reply:
`builtin_backends` is a list of backends included in the exception message. We want to omit packages that are not a valid option, `postgresql_psycopg2` doesn't exist anymore so there is no point in including it here.